docs/copilot/copilot-settings.md

```diff
@@ -0,0 +1,88 @@
+| `setting(chat.experimental.detectParticipant.enabled)` <i class="codicon codicon-beaker"></i> | false  | Enable chat participant detection in the Chat view.   |
+
+## Inline chat settings
+
+| Setting | Default | Description |
+|--------|----------|-------------|
+| `setting(inlineChat.acceptedOrDiscardBeforeSave)` | true | Controls whether pending inline chat sessions in an editor prevent saving the file.  |
+| `setting(inlineChat.finishOnType)` | `off`  | Whether to finish an inline chat session when typing outside of changed regions.  |
+| `setting(inlineChat.holdToSpeech)` | true  | Whether holding the inline chat keybinding will automatically enable speech recognition.  |
+| `setting(inlineChat.mode)` | `live`  | Configure if changes crafted with inline chat are applied directly to the document or are previewed first.  |
+
+## Notebook settings
+
+| Setting | Default | Description |
+|--------|----------|-------------|
+| `setting(notebook.experimental.generate)` <i class="codicon codicon-beaker"></i> |  true | Enable the **Generate** action to create code cells with inline chat enabled in the notebook editor. |
+
+## Accessibility settings
+
+| Setting | Default | Description |
+|--------|----------|-------------|
+| inlineChat.accessibleDiffView |  `auto` | Whether the inline chat also renders an accessible diff viewer for its changes.  |
+| `setting(accessibility.signals.chatRequestSent)` | `{ "sound": "auto", "announcement": "auto" }` | Plays a signal - sound (audio cue) and/or announcement (alert) - when a chat request is made.  |
+| `setting(accessibility.signals.chatResponseReceived)` | `{ "sound": "auto" }` | Plays a sound / audio cue when the response has been received.  |
+| `setting(accessibility.verbosity.inlineChat)` | true | Provide information about how to access the inline editor chat accessibility help menu and alert with hints that describe how to use the feature when the input is focused.   |
+| `setting(accessibility.verbosity.inlineCompletions)` | true | Provide information about how to access the inline completions hover and Accessible View.  |
+| `setting(accessibility.verbosity.panelChat)` | true | Provide information about how to access the chat help menu when the chat input is focused.  |
+| `setting(accessibility.voice.keywordActivation)` | `off` | Controls whether the keyword phrase 'Hey Code' is recognized to start a voice chat session.  |
+| `setting(accessibility.voice.autoSynthesize)` | `off` | Controls whether a textual response should automatically be read out aloud when speech was used as input. |
+| `setting(accessibility.voice.speechTimeout)` | 1200 | The duration in milliseconds that voice speech recognition remains active after you stop speaking.  |
+
+## Related resources
+
+* [Get started with the Copilot in VS Code Quickstart](/docs/copilot/getting-started.md)
+* [Get a quick overview of the Copilot features in VS Code](/docs/copilot/copilot-vscode-features.md)
```

Comment:
These two links seem to mean the same thing at first glance. Could there be wording to distinguish these two, or could just one link be shown?